src/bytes.ts

```diff
@@ -34,34 +35,62 @@ export const setLengthLeft = function(msg: any, length: number, right: boolean =
     return msg.slice(-length)
   }
 }
-export const setLength = setLengthLeft
 
 /**
- * Right Pads an `Array` or `Buffer` with leading zeros till it has `length` bytes.
+ * Right Pads a `Buffer` with leading zeros till it has `length` bytes.
  * Or it truncates the beginning if it exceeds.
- * @param msg the value to pad (Buffer|Array)
+ * @param msg the value to pad (Buffer)
  * @param length the number of bytes the output should be
- * @return (Buffer|Array)
+ * @return (Buffer)
  */
-export const setLengthRight = function(msg: any, length: number) {
-  return setLength(msg, length, true)
+export const setLengthRight = function(msg: Buffer, length: number) {
+  return setLengthLeft(msg, length, true)
 }
 
 /**
- * Trims leading zeros from a `Buffer` or an `Array`.
+ * Trims leading zeros from a `Buffer`.
+ * @param a (Buffer)
+ * @return (Buffer)
+ */
+export const unpadBuffer = function(a: any): Buffer {
```

Comment:
I assume having an `any` type here is a mistake?

Reply:
Ai! Thanks.

Reply:
Addressed in 1f2803c